custom_navbutton_completion.js

```diff
@@ -0,0 +1,119 @@
+/**
+ * Created by ishani on 01/03/2017.
+ */
+
+custom_navbutton_completion = {
```

Comment:
At the very least, this should be (even if you don't package as a YUI module):
M.block_navbuttons = M.block_navbuttons || {};
M.block_navbuttons.completion = {